CenterForOpenScience/isp
# Add German translation

Refs: https://openscience.atlassian.net/browse/LEI-324

## Purpose
Add German translation (de) to ISP

## Summary of changes
- Add german translation
- Add consent text for german-speaking sites
- Fix the script that downloads consent forms
- Update misc doc and formatting code based on first translation in the wild

## Testing notes

## What to test
- Pick German as your language. Move through the study and confirm that all text is translated.
- Use German as your language, but with an English-speaking site ID. Confirm that the consent form is in english but the study is in German.
  - ...and vice versa
- Verify the correct consent form text appears for one account created for each site ID

## Where to test
This affects the following choices in the language picker:
- Austria (deutsch)
- Germany (deutsch)
- Switzerland (deutsch)

All three language codes will use the same underlying `de` translation.

The consent text is controlled separately from the translation, and is controlled by the study ID. The following individual consent forms should be verified by creating accounts for the appropriate study IDs (study IDs are case sensitive)
- BASE.CH
- BERL.DE
- GOTT.DE
- GRAZ.AU
- INNS.AU
- ZURI.CH
